ot/mpz-ot/src/kos/receiver.rs

```diff
@@ -188,6 +141,70 @@ where
     type Msg = Message<BaseOT::Msg>;
 }
 
+#[async_trait]
+impl<BaseOT> OTSetup for Receiver<BaseOT>
+where
+    BaseOT: OTSetup + OTSender<[Block; 2]> + Send,
+{
+    async fn setup<
+        Si: IoSink<Message<BaseOT::Msg>> + Send + Unpin,
+        St: IoStream<Message<BaseOT::Msg>> + Send + Unpin,
+    >(
+        &mut self,
+        sink: &mut Si,
+        stream: &mut St,
+    ) -> Result<(), OTError> {
+        if self.state.is_extension() {
+            return Ok(());
```

Comment:
In what situations will it make sense for KOS to re-use baseOT delta?  I cant think of any. Maybe we should only allow calling `setup()` in `state.is_initialized()` ?